circle.yml

```diff
@@ -18,12 +18,10 @@ jobs:
       - run: set -o pipefail &&
              swiftlint lint --strict --reporter json |
              tee $CIRCLE_ARTIFACTS/swiftlint-report.json
-      - run: bin/test iOS 10.3.1
       - run: bin/test iOS 11.2
```

Comment:
`11.4`?

Reply:
CircleCI only has 11.2 sim, I've updated the min deployment target to 11.4.